package.json

```diff
@@ -51,6 +54,10 @@
     "typescript": "4.1.3",
     "genversion": "3.0.2"
   },
+  "resolutions": {
+    "**/@types/react": "^16.13.1",
+    "**/@types/react-native": "0.63.42"
```

Comment:
Is this a correct version? I can see only version `0.63.4` for RN, but not `0.63.42` (but maybe types package has a different versioning).

Reply:
yes this is the version for the types package